app/graphql/mutations/create_dynamic_field_group.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Mutations::CreateDynamicFieldGroup < Mutations::BaseMutation
+  argument :string_key, ID, required: true
+  argument :display_label, String, required: true
+  argument :sort_order, Integer, required: true
```

Comment:
`sort_order` and `is_repeatable` can be optional here.

Reply:
If we are allowing sort_order as an argument, then the resolve method will probably need to handle updating the sort_order of adjacent elements.  As in:

field_a (order 1)
field_b (order 2)
field_c (order 3)

running this: create field_z with order 2
should result in updates to the order values of field_b and field_c:

field_a (order 1)
field_z (order 2)
field_b (order 3)
field_c (order 4)

Reply:
Last time we talked about this, we talked about just doing separate requests if we wanted to update the order of a whole list of elements.